### ISSUE TITLE
Does not work in Vim

### ISSUE THREAD

**Issue by rkvirani** [Shadorain/shadotheme#11]
Tried in stock vim 8.2 and Vim 9.0

Error detected while processing /home/user/dotfiles/.vim/plugged/shadotheme/colors/shado-legacy.vim:
line   56:
E423: Illegal argument: blend=10

**Comment by Shadorain** [comment]
Hmm, maybe the `blend` doesn't work in vim 🤔  I haven't actually tested on vim since switching a few years back. If you take out all the `blend`s I wonder if it will work?
Sorry for this inconvenience and thank you for opening an issue!

**Comment by CodeConnoisseur74** [comment]
Works for me in Vim. Need to call shado-legacy . Thanks for adding the shado-legacy.vim. Much appreciated!

**Comment by Shadorain** [comment]
Yes I added shado-legacy.vim to work on vim. It isnt updated though whereas if you switch to Neovim you will have the updated shadotheme.

**Comment by CodeConnoisseur74** [comment]
@Shadorain The updated version does work in Neovim and not updated in Vim. Yuo are correct. :)
